include/ignition/gazebo/gui/GuiSystem.hh

```diff
@@ -43,8 +41,6 @@ namespace gazebo
   /// separate process that is stepped by updates coming through the network
   class GuiSystem : public ignition::gui::Plugin
   {
-    Q_OBJECT
```

Comment:
Just checking, is this not needed because `ignition::gui::Plugin` already has it?

Reply:
If my memory serves me well I removed it since it does not fit into `Notice that the Q_OBJECT macro is mandatory for any object that implements signals, slots or properties` described here https://doc.qt.io/qt-5/qobject.html

Reply:
See this:

_We strongly recommend the use of this macro in all subclasses of QObject regardless of whether or not they actually use signals, slots and properties, since failure to do so may lead certain functions to exhibit strange behavior._

We're using the fact that this is a `QObject` in places like this:

https://github.com/ignitionrobotics/ign-gazebo/blob/8fd9b2647a5c37b1dde5509b5a741c6c955ee10f/src/gui/GuiRunner.cc#L93

So I think it should be left there, but I don't know what problem that was causing on Windows.

Reply:
79dff1c reverted.